lib/sgtn-client/util/service-util.rb

```diff
@@ -0,0 +1,99 @@
+# 
+#  Copyright 2019-2022 VMware, Inc.
+#  SPDX-License-Identifier: EPL-2.0
+#
+
+require 'multi_json'
+
+module SgtnClient
+
+  # A service util class to serve for translation API
+  class ServiceUtil
+    class << self
+
+      # Get key_translation
+      def get_translation(component, key, locale)
+        locale = SgtnClient::LocaleUtil.get_best_locale(locale)
+        items = get_cs(component, locale)
+        if items.nil? || items["messages"].nil?
+          nil
+        else
+          items["messages"][key]
+        end
+      end
+
+      # Compare local source with remote source
+      def compare_source(component, key, source_locale, source, translation)
```

Comment:
Why do you have to pass source_locale here? Just use `SgtnClient::LocaleUtil.get_source_locale` inside compare_source.

Reply:
agree